src/rh-footer-links-wrapper.ts

```diff
@@ -36,90 +39,93 @@ export class RhFooterLinkWrapper extends LitElement {
 
       :host([is-mobile]) {
         --link-font-size: 16px;
+        --rh-footer--links-columns: 1;
```

Comment:
`--` everywhere or `-` only

my preference is for `-`

Reply:
+1 for only `-`.  What about prefixing?  I worry about the generic names `link-font-size` but I would do prefer non-prefixed.

Reply:
that's a broader discussion with @coreyvickery I think

Ideally this would be some theme property or something

if it's undocumented "fake private" I'm less bothered

probably prefix for now until we achieve further clarity

Reply:
going to put this as a todo right now.